fcrepo-kernel-modeshape/src/test/java/org/fcrepo/integration/kernel/modeshape/FedoraResourceImplIT.java

```diff
@@ -752,6 +755,8 @@ public void testReplacePropertiesHashURIs() throws RepositoryException {
         final Model updatedModel3 = object.getTriples(subjects, PROPERTIES).collect(toModel());
 
         updatedModel3.remove(subject, dcCreator, hashResource);
+        updatedModel3.removeAll(hashResource, (Property)null, (RDFNode)null);
```

Comment:
I am happy to remove it as a part of this PR, unless there is some reason for this update that I am missing.